components/Eingabefeld 1/index.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import { locations } from "../../db/data.js";
+import moment from "moment";
+
+const now = moment();
+const lockedTime = now.set().format("HH:mm");
+
+export default function InformationPage() {
+  return (
+    <OverAllWrapper>
```

Comment:
The name is not very descriptive and can easily be confused, better here again think of a new name.